bundles/org.openhab.binding.shelly/src/main/resources/ESH-INF/config/config.xml

```diff
@@ -39,8 +45,71 @@
 			<label>Enable Push Events (short/long)</label>
 			<description>True if the binding should register to get the Short/Long Push Events.</description>
 			<advanced>true</advanced>
+			<default>false</default>
+		</parameter>
+		<parameter name="eventsCoIoT" type="boolean" required="false">
+			<label>Enable CoIoT Events</label>
+			<description>If enabled the Coap protocol will be used to receive status updates.</description>
```

Comment:
```suggestion
			<description>If enabled, the CoAP protocol is used to receive status updates.</description>
```

Reply:
changed

Reply:
did you maybe miss to push your changes?